packages/frontend/src/components/SettingsValidator/ValidatorTable.tsx

```diff
@@ -51,7 +51,7 @@ export const ValidatorTable: FC<{
     data: ValidationResponse[];
     projectUuid: string;
 }> = ({ data, projectUuid }) => {
-    const { classes } = useTableStyles();
+    const { cx, classes } = useTableStyles();
```

Comment:
that's where you get the `cx`!! 🤦  I didn't know 😆

Reply:
it is well hidden!!!